src/Snppts/Snppts.csproj

```diff
@@ -24,7 +24,5 @@
     <Folder Include="Views\Shared\Components\PopularCategories\" />
     <Folder Include="Views\Shared\Components\AllCategories\" />
     <Folder Include="Views\Shared\Components\RandomSnippets\" />
-	<Folder Include="Snippets\" />

```

Comment:
Can you undo the changes in the csproj file?

Reply:
yeap, i will remove.

Reply:
in the fork of the project is it not visible to me, something different to do?

Reply:
That’s the problem. These lines are removed while they should still be there :)

Reply:
humm, sorry, is it necessary to send a new PR?

Reply:
It should not! You can either revert this change in a new commit or simply add back these lines in a new commit

Reply:
added again and made a new commit